languages/python/sqlalchemy-oso/tests/test_post_relationship.py

```diff
@@ -396,6 +396,49 @@ def test_partial_in_collection(session, oso, tag_nested_many_many_test_fixture):
     assert len(posts) == 1
 
 
+def test_empty_constraints_in_becomes_count_gt_0(
```

Comment:
Maybe rename this test in both SQLAlchemy & Django so the test name is agnostic to the query (`becomes_count_gt_0` portion).  SQLAlchemy doesn't translate to that.

Reply:
afc1c66